docs/titan-basics.md

```diff
@@ -89,7 +89,10 @@ the first time until you have installed the Titan PC Suite software,
 which you do simply by running the installer file. This will make sure
 the drivers for the panel are installed.
 
-> From Titan v7, Windows XP and Vista are no longer supported. Your PC must be running a 64 bit version of Windows 7, Windows 8.1 or Windows 10.
+> From Titan v13, your PC must be running a 64 bit version of Windows 10. 
+  Earlier versions of Windows and virtual machines are not supported.
```

Comment:
Probably worth saying "are not supported and are used at your own risk" to give people a hint that it is possible

Reply:
Is that VM's are possible but not supported? Or earlier windows versions may also work? Kade's comment seemed to say that other windows versions definitely don't work.

Reply:
VMs are possible, lots of users do it we just don't provide them with support if anything breaks.

On the version of windows, technically others will still work right now but I'd rather just say we only support windows 10 64 bit because that will definitely be the case in future.